webpack.config.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+
+const path = require('path')
+const HtmlWebPackPlugin = require('html-webpack-plugin')
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin')
+const PACKAGE = require('./package.json')
+const version = PACKAGE.webpack_bundle_version
+const CleanWebpackPlugin = require('clean-webpack-plugin')
+const project_config = require('./config.js')
+
+const fs = require('fs')
+
+const pathsToClean = [
+  'dist',
+]
+
+function generateHtmlPlugins (templateDir) {
+  const templateFiles = fs.readdirSync(path.resolve(__dirname, templateDir))
+  
+  // console.log(templateFiles)
+  return templateFiles
+    .filter(item => item.includes('.hbs'))
+    .map(item => {
+      // Split names and extension
+      const parts = item.split('.')
+        const name = parts[0]
+        const extension = parts[1]
+        return new HtmlWebPackPlugin({
+          template: path.resolve(__dirname, `${templateDir}/${name}.${extension}`),
+          filename: `${name}.html`,
+          mobile: true,
+          site: {
+            title: project_config.title,
+            description: project_config.desctiption,
+            baseurl: project_config.description,
+            url: project_config.url,
+            custom_class: project_config.custom_class
+          },
+          tracking: {
+            google_analytics_id: project_config.google_analytics_id,
+            twitter_id: project_config.twitter_id,
+            facebook_id: project_config.facebook_id
+          }
+        })
+    })
+}
+
+const htmlPlugins = generateHtmlPlugins('./src')
```

Comment:
Rather than defining this here, it could just called inline on line #121 since it's only referenced once.